js/core/clusters/clusters.js

```diff
@@ -53,6 +53,11 @@ define([
   return function(config) {
     var lock = false;
 
+    // getter for config.cluster
+    window.cluster = function() {
```

Comment:
I did not say that config should removed, needless to convince me about its existence. I'm just talking about config.cluster here.

As soon as there are 2 ways to access the same information and it introduces confusion into other developers mind, it's a technical debt.